cmake/procstat-config.cmake

```diff
@@ -0,0 +1,27 @@
+include(FindPackageHandleStandardArgs)
+include("$ENV{WORKSPACE_TOP}/common/cmake/lb.cmake")
+
+lb_local_path(PROCSTAT_BUILD procstat procstat)
```

Comment:
@elazar-lb this can't go upstream, right?
btw, why the user of this cmake script can't provide procstat install path via `CMAKE_PREFIX_PATH` or some other environment variable?

Reply:
Right, it's our lb_build.

Not easy/good idea to change CMAKE_PREFIX_PATH from a module.

PS
"'"upstream"""